content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/100-special-rules-for-referential-actions.mdx

```diff
@@ -1,15 +1,17 @@
 ---
 title: 'Special rules for referential actions in SQL Server'
-metaTitle: 'Special rules for referential actions in SQL Server'
-metaDescription: 'Circular references or multiple cascade paths can cause validation errors on Microsoft SQL Server. Since the database does not handle these situations out of the box, learn how to solve this problem.'
+metaTitle: 'Special rules for referential actions in SQL Server and MongoDB'
+metaDescription: 'Circular references or multiple cascade paths can cause validation errors on Microsoft SQL Server and MongoDB. Since the database does not handle these situations out of the box, learn how to solve this problem.'
 tocDepth: 2
 ---
 
 <TopBlock>
 
-Microsoft SQL Server doesn't allow cascading [referential actions](./) on a foreign key, if the relation chain causes a cycle or multiple cascade paths. Most SQL databases handle these references out of the box.
+Some databases have specific requirements that you should consider if you are using referential actions.
 
-If the referential actions on the foreign key are set to something other than `NO ACTION`, the server will check for cycles or multiple cascade paths and return an error when executing the SQL.
+- Microsoft SQL Server doesn't allow cascading [referential actions](./) on a foreign key, if the relation chain causes a cycle or multiple cascade paths. If the referential actions on the foreign key are set to something other than `NO ACTION` (or `NoAction` if Prisma is managing referential integrity), the server will check for cycles or multiple cascade paths and return an error when executing the SQL.
+
+- with MongoDB, using referential actions in Prisma requires that for any data model with self-referential relations or cycles between three models, you must set the referential action of `NoAction` to prevent the referential action emulations from looping infinitely. Be aware that by default, the `referentialIntegrity = "prisma"` mode is used for MongoDB, which means that Prisma manages [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity)
```

Comment:
```suggestion
- with MongoDB, using referential actions in Prisma requires that for any data model with self-referential relations or cycles between three models, you must set the referential action of `NoAction` to prevent the referential action emulations from looping infinitely. Be aware that by default, the `referentialIntegrity = "prisma"` mode is used for MongoDB, which means that Prisma manages [referential integrity](/concepts/components/prisma-schema/relations/referential-integrity).
```

Reply:
@Jolg42 your edit does sound better is some ways (if spoken for example), but in tech docs we do wants to use the simple present tense form of "manages." @matthewmueller your choice, as author, but I'd vote for leaving it as you had it. :-)